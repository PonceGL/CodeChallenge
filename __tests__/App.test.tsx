import "react-native";
import React from "react";
import App from "../App";

import renderer, { ReactTestRenderer } from "react-test-renderer";

describe("App", () => {
  const element = (): ReactTestRenderer => {
    const app = renderer.create(<App />);
    return app;
  };

  test("should render correctly", () => {
    const app: ReactTestRenderer = element();
    const tree = app.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
