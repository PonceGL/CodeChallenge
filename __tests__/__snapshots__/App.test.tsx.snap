// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<View
  accessibilityLabel="Pantalla de inicio"
  style={
    [
      {
        "flex": 1,
      },
      {
        "backgroundColor": "#F8F8F8",
      },
    ]
  }
>
  <RCTSafeAreaView
    style={
      {
        "flex": 1,
      }
    }
  >
    <View
      style={
        {
          "borderColor": "#3E54AC",
          "borderWidth": 2,
          "flex": 1,
        }
      }
    >
      <Text>
        Home
      </Text>
    </View>
  </RCTSafeAreaView>
</View>
`;
